tardis/io/atom_data/base.py

```diff
@@ -392,6 +392,9 @@ def prepare_atom_data(
             ].copy()
 
             if line_interaction_type == "downbranch":
+                logger.debug(
+                    "Line Interaction Type for the Photons are DownBranch, Proceeding with Preparation of Atom Data"
```

Comment:
You are capitalizing words that are not variables. I think it would be best to move from `Title Case` to `Sentence case`.

Reply:
Should this be done throughout the PR or this specific instance?

Reply:
Resolved, making the required changes 😉